frontend-svelte/src/routes/+layout.ts

```diff
@@ -1 +1,25 @@
+import type { LayoutLoad } from "./$types";
+import { getMe } from "$lib/api/users";
+import { redirect } from '@sveltejs/kit';
+import { UNAUTHORIZED_ERROR } from '$lib/api/model/error';
+import { base } from '$app/paths';
+import { error } from '@sveltejs/kit';
+
 export const ssr = true;
+
+export const load: LayoutLoad  = async ({ url, fetch, }) => {
+    if (url.pathname === `${base}/login`) {
+		// no redirect if user already on login page
+        return {};
+    }
+    try {
+		const me = await getMe(fetch);
+		return { me, meError: null };
+	} catch (e) {
+		if (e === UNAUTHORIZED_ERROR) {
+			redirect(302, `${base}/login`);
+		}
+		// if error is other than UNAUTHORIZED_ERROR, backend is in unexpected state
+		error(500, "Unexpected server response while attempting to fetch user data from backend")
```

Comment:
What I've optimized for with error handling, so far, is that users can send me a screenshot of their issues and I can directly see what's going on (without having them open dev tools first, ...). So, I'd also try to display the actual status code and body sent by the backend/received by the frontend.

(can be follow-up PR)